python/sglang/srt/model_executor/model_runner.py

```diff
@@ -186,6 +186,9 @@ def init_torch_distributed(self):
         elif self.device == "xpu":
             torch.xpu.set_device(self.gpu_id)
             backend = "gloo"
+        elif self.device == "hpu":
+            torch.get_device_module(self.device).set_device(self.gpu_id)
+            backend = "hccl"
```

Comment:
Refine other device path with "torch.get_device_module(self.device).set_device(self.gpu_id)" may be a better choice to make the code simplifier.

Reply:
done , this will be further simplified once https://github.com/pytorch/pytorch/pull/140536 is available, that way we dont have any device name references in the file.